pkg/audit/verifier.go

```diff
@@ -176,6 +176,11 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe, skip map[s
 	for _, pieceNum := range pieceNums {
 		failedNodes = append(failedNodes, shares[pieceNum].NodeID)
 	}
+	//remove failed audit pieces from the pointer so as to only penalize once for failed audits
```

Comment:
nit: add a space between `//` and `remove`
nit(2): Add an empty line above the comment.